src/web/history.rkt

```diff
@@ -147,3 +147,107 @@
                         "\\[\\leadsto "
                         ,(core->tex step-prog #:loc (cons 2 loc) #:color "blue")
                         "\\]"))))))))
+
+
+(define (render-json altn pcontext pcontext2 ctx)
+  (define repr (context-repr ctx))
+  (define err
+    (format-accuracy (errors-score (errors (alt-expr altn) pcontext ctx)) (representation-total-bits repr) #:unit "%"))
+  (define err2
+    (format-accuracy (errors-score (errors (alt-expr altn) pcontext2 ctx)) (representation-total-bits repr) #:unit "%"))
```

Comment:
I think in JSON we might as well _not_ format the accuracy and just put `total-bits` and `errors-score` both in the hash.